### ISSUE TITLE
can we enable direct_io flag when issue file system API from MLPerf storage test suite

### ISSUE THREAD

**Issue by gaowayne** [mlcommons/storage#51]
currently, we notice, MLPerf storage still leverage page cache a lot, not stress storage enough. 
can we support one branch that issue direct IO for all places? 
I see there are too many file open places.

**Comment by LouisDDN** [comment]
I don't think it makes sense as the goal is to reproduce a workload not to get maximum throughput out of the storage.
